src/Product/Image/Assignor.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
@kuiche I'm going to take a call on this and say that the class name is okay since it'd be really good to get this feature rolled out